duckduckgo/macos-browser
# Show preferences instead of Bitwarden app in case of communication error

Task/Issue URL: https://app.asana.com/0/1177771139624306/1204302365133777/f
CC:

**Description:**
If autofill message says "Bitwarden is locked" but communication is broken, it's more useful to navigate to our Autofill Settings instead of Bitwarden app. The Settings page contains an error message and provides an option to go through the onboarding process again.

**Steps to test this PR:**

1. Go to Settings -> Autofill and set Bitwarden as password manager
3. Finish the onboarding flow and make sure the autofill of Bitwarden passwords work
5. Quit Bitwarden app
7. Try to use autofill again and make sure "Bitwarden is locked" autofill message opens Bitwarden after clicking on it
9. Quit DuckDuckGo app
11. Open Keychain and remove com.duckduckgo.macos.browser.debug.bitwarden.shared.key 
13. Open DuckDuckGo app again
15. Try to use autofill again but this time "Bitwarden is locked" autofill message must open Settings -> Autofill
17. Make sure the Settings page contains error message
